packages/components/src/components/hds/dialog-primitive/header.ts

```diff
@@ -24,6 +27,15 @@ export interface HdsDialogPrimitiveHeaderSignature {
 const NOOP = (): void => {};
 
 export default class HdsDialogPrimitiveHeaderComponent extends Component<HdsDialogPrimitiveHeaderSignature> {
+  /**
```

Comment:
We opted to discontinue the jsdoc style comments, so you can remove these and don't need to add them in the future.